### ISSUE TITLE
[BUG] ng-formio no longer contains formio-complete.js

### ISSUE THREAD

**Issue by PurpleDuckMedia** [formio/ngFormio#663]
Currently using Angular JS 1.72 and ng-formio 2.35.3. Trying to upgrade to a newer version so we use the TemplateAPI, however, the new version of ng-formio v4.7.9 no longer contains form-complete.js

The documentation refers to 

https://rawgit.com/formio/ngFormio/master/dist/formio-complete.min.js

But getting a 404 when including that

**Comment by randallknutson** [comment]
ng-formio is much simpler than it was on the older versions since most of the logic and functionality has moved to formio.js. Because of this you sholud be able to use https://github.com/formio/ngFormio/blob/master/dist/ng-formio.min.js instead of the formio-complete from the previous library.

**Comment by PurpleDuckMedia** [comment]
Hi Randal, I still can't get this to work. I created a quick code pen just to make sure it wasn't any of the other code in the solution.

https://codepen.io/purpleduckmedia/pen/VwYBBXW

But this fails to load fomio module. Which I assume used to be in formio-complete.min.js. I did see on your example page this it loads 

https://formio.github.io/ngFormio/dist/formio-full.js

But the issue with this is that contains angular and as part of our app we need to load that separately.
